controllers/channel.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const mysql   = require('anytv-node-mysql');
+const winston = require('winston');
+const util = require(__dirname + '/../helpers/util');
+
+
+exports.get_channels = (req, res, next) => {
+    let connection = mysql.use('dashboard_db');
+    let data = util.get_data({
+        'user_id': '',
+        '_page': 0,
+        '_limit': 0
+    }, req.query);
+    let page = data.page || 1;
```

Comment:
data can be an error if user_id is missing. do this on start
data.page = data.page || 1